charts/alertmanager-snmp-notifier/values.yaml

```diff
@@ -0,0 +1,185 @@
+restartPolicy: Always
```

Comment:
please sort all vars alphabetically.

Reply:
Fixed, although I kept the `helm create` order if that is ok for you.